README.md

```diff
@@ -44,27 +44,20 @@ When launching the app, you can specify the port to use with a command like the
 ```
 ./gradlew bootRun -Pport=8888
 ```
-### Configuring the settings
-There are three ways for user to configure the settings. 
+### Configuring application-related properties
+1. When running the application from sources:
+   
+    Change the [application.properties](https://github.com/parasoft/parasoft-demo-app/blob/main/src/main/resources/application.properties) file directly or put a customized version of the file to overwrite the properties and put it under config folder before running the application.
 
-1. Change the [**application.properties**](https://github.com/parasoft/parasoft-demo-app/blob/main/src/main/resources/application.properties) file directly in the original code.
-2. Create a custom application.properties file.
-* When launching the app as a .war file, put it in the {The sibling directory of .war file}/config.
-* When launching the app directly from sources, put it in the {The root directory of project}/config.
+    Notes: all configurable properties are available in [the example version of application.properties](https://github.com/parasoft/parasoft-demo-app/blob/main/application.properties).
```

Comment:
I'm not sure whether it's a need to add a link for the example version of application.properties file.
If need, it will work after this branch is pused to master.